ui/components/multichain/network-list-menu/network-list-menu.js

```diff
@@ -80,14 +83,44 @@ export const NetworkListMenu = ({ onClose }) => {
 
   const showSearch = nonTestNetworks.length > 3;
 
+  const orderedNetworksList = useSelector(getOrderedNetworksList);
+
+  const newOrderNetworks = () => {
```

Comment:
Can this be a new selector?  I feel like we probably don't want to be doing data management in the components themselves.

Reply:
Hmm...I am not sure if we need a selector for this as we need this update only in this file.